apiary.apib

```diff
@@ -3289,6 +3289,144 @@ Download a credit note in a specific format.
             + location: `https://cdn.teamleader.eu/file` (string) - A temporary url where the requested file can be downloaded
             + expires: `2018-02-05T16:44:33+00:00` (string) - Expiration time of the temporary download link
 
+## Subscriptions [/subscriptions]
+
+### subscriptions.list [GET /subscriptions.list]
+
++ Request (application/json)
+    + Attributes (object)
+        + filter (object, optional)
+            + ids: `46156648-87c6-478d-8aa7-1dc3a00dacab`, `46156648-87c6-478d-8aa7-1dc3a00daca4` (array[string], optional)
+            + invoice_id: `60f03745-4289-4ae7-8c74-d797860bc887` (string, optional) - Filter on subscriptions created from an invoice
+            + deal_id: `fddf42f9-bda1-0c49-a22b-0bf147cd1e3f` (string, optional) - Filter on subscriptions created from a deal
+            + customer (Customer)
+        + page (Page, optional)
+        + sort (array, optional)
+            + (object)
+                + field (enum, required)
+                    + Members
+                        + title
+                        + created_at
+                + order (enum, optional)
+                    + Members
+                        + asc
+                        + desc
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `e2314517-3cab-4aa9-8471-450e73449041` (string)
+                + title: `Subscription for cookies` (string)
+                + note: `'Some extra remarks about the subscription'` (string, nullable) - plaintext
+                + state: `active` (enum[string])
+                    + Members
+                        + active
+                        + deactivated
+                + department (object)
+                    + type: `department` (string)
+                    + id: `5e90eb0a-b502-4344-aa0f-3b8525af6186` (string)
+                + invoicee (object)
+                    + customer (Customer)
+                    + for_attention_of (object, nullable)
+                        + contact (object, nullable)
```

Comment:
Q: Is this nullable? Will it not always be set if for_attention_of is not null?

Reply:
Yes the FAO has an optional contact.